src/scenes/NftDetailPage/NftAdditionalDetails/NftAdditionalDetailsTezos.tsx

```diff
@@ -0,0 +1,114 @@
+import { graphql, useFragment, useLazyLoadQuery } from 'react-relay';
```

Comment:
the only two things different in this file than the regular eth one is

- View on Objkt
- Contract address on tzkt
- TezosDomain component